packages/site-admin/src/components/page/style.scss

```
@@ -0,0 +1,64 @@
+@import "@wordpress/base-styles/variables";
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/z-index";
+
+.a8c-site-admin-page {
+	color: $gray-800;
+	background: $white;
+	height: calc(100% - #{$header-height});
+	container: a8c-site-admin-page / inline-size;
+
+	@media not (prefers-reduced-motion) {
+		transition: width ease-out 0.2s;
+	}
+
+	@include break-medium() {
+		height: 100%;
+	}
+}
+
+.a8c-site-admin-page-header {
+	padding: $grid-unit-20 $grid-unit-60;
+	border-bottom: 1px solid $gray-100;
+	background: $white;
+	position: sticky;
+	top: 0;
+	z-index: 2; // hardcoded for now ARC-95
+
+	@media not (prefers-reduced-motion) {
+		transition: padding ease-out 0.1s;
+	}
+
+	.components-heading {
+		color: $gray-900;
+	}
+
+	.a8c-site-admin-page-header__page-title {
+		min-height: $grid-unit-50;
+
+		.components-heading {
+			flex-grow: 1;
+			flex-basis: 0;
+			white-space: nowrap;
+		}
+	}
+
+	.a8c-site-admin-page-header__sub-title {
```

Given this is a copy we could leave this as is, but weird we are not making use of `&` here same for the `page-title`
```suggestion
	&__sub-title {
```